src/Services/RabbitMQService.cs

```diff
@@ -33,29 +31,13 @@ public RabbitMQService(string connectionString, string hostName, string userName
             _model = connectionFactory.CreateConnection().CreateModel();
         }
 
-        public RabbitMQService(string connectionString, string hostName, string queueName, string userName, string password, int port, string deadLetterExchangeName)
+        public RabbitMQService(string connectionString, string hostName, string queueName, string userName, string password, int port)
             : this(connectionString, hostName, userName, password, port)
         {
             _rabbitMQModel = new RabbitMQModel(_model);
-
-            _deadLetterExchangeName = deadLetterExchangeName;
             _queueName = queueName ?? throw new ArgumentNullException(nameof(queueName));
 
-            Dictionary<string, object> args = new Dictionary<string, object>();
-
-            // Create dead letter queue
-            if (!string.IsNullOrEmpty(_deadLetterExchangeName))
-            {

```

Comment:
We should allow users to specify a `deadLetterExchangeName` . I think you just have to delete creating the queue here. Also,  log an error if adding message to `deadLetterExchangeName` fails